core/test/unit/lib/request_spec.js

```diff
@@ -133,7 +133,8 @@ describe('Request', function () {
 
         const requestMock = nock('http://nofilehere.com')
             .get('/files/test.txt')
-            .reply(500, {message: 'something aweful happend', code: 'AWFUL_ERROR'});
+            .times(3)   // 1 original request + 2 default retries
```

Comment:
@gargol Was trying to understand from changelog/commit, is the `times` here with the value representing default, which is it will 3 times before returning 500? If so, is it possible to override it to use `1` instead of `3` before returning the status code? Also, any reason we skipped sending the message along with status code below, as in old code?

Reply:
This part is configuring nock (mocking external server that is pinged by `got`), so it had to adjust to the changes to the new default behavior of got (now it makes 2 additional retries when 500, and [bunch others](https://github.com/sindresorhus/got/commit/a3e77de2876c2594af797cba250090b70735c8db#diff-782a39c75a473c55bd4c08418d38e5daR69), is returned). If we just return 500 after 1st request, got will not be able to reach anything on the retry. 

Think the question is more if we should allow making 2 additional requests for your wrapper of got.

Reply:
Ah ok, got it. Yeah i don't see any bad implication either, and seems to be a sane default. Either ways, we should still be adding the message along with status code here in tests like [previously](https://github.com/TryGhost/Ghost/pull/10861/files#diff-bbd12322a1f4631305996c4a89a8ef2eL136), unless i am missing something.

Reply:
@rishabhgrg have added the response body back to the test and also a check in the test itself. Merging it :ship: